packages/aws-cdk-lib/aws-codepipeline/README.md

```diff
@@ -365,6 +365,8 @@ const replicationBucket = new s3.Bucket(replicationStack, 'ReplicationBucket', {
 
 ## Variables
```

Comment:
Lets have a few sentences here describing what the different scopes mean for variables

Reply:
https://github.com/aws/aws-cdk/pull/28538/commits/f20a00cb4110265e3254cd1ae250c9c1b73799ea